spec/features/support/providers/accredited_provider_search/searching_for_an_accredited_provider_spec.rb

```diff
@@ -60,6 +80,10 @@ def when_i_select_the_provider
     click_continue
   end
 
+  def and_i_continue_without_entering_a_description
```

Comment:
Can we use `alias_method :and_i_continue_without_entering_a_description, :click_continue`

Instead of re-defining a new method?